index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+
+  <body>
+    <header id="header">
+      
+      <nav id="nav-bar">
+        <a href="#html" class="nav-link">HTML</a> |
+        <a href="#css" class="nav-link">CSS</a> |
+        <a href="#JS" class="nav-link">JavaScript</a> |
+        <a href="#python" class="nav-link">Python</a>
+      </nav>
+      <img  id="header-img" src="https://res.cloudinary.com/growsurf-prod/image/upload/v1582211139/production/gnysw2objzekbagrqiax.png" alt="Image">
+      <br><br><br>
+
+    </header>
+    
+    <br><br><br>
+    <div id="html">
+      
+        <p>HTML is a programming language that is widely used</p>
+
+    </div>
+    <br><br><br>
+    <video id="video" width="500" height="500" controls>
```

Comment:
- [ ] Please avoid usage of íds in your tags instead use classes

Reply:
@ZawadiSumba66 Please explain why the class usage is preferred for example:
 (the usage of classes will make it easy to follow the DRY principle and the style can applied to other elements that have the same class so it's time saving as well)